tests/sentry/tasks/test_post_process.py

```diff
@@ -1752,9 +1752,11 @@ def test_forecast_in_activity(self, mock_is_escalating):
         """
         event = self.create_event(data={"message": "testing"}, project_id=self.project.id)
         group = event.group
-        group.status = GroupStatus.IGNORED
-        group.substatus = GroupSubStatus.UNTIL_ESCALATING
-        group.save()
+        group.update(
+            first_seen=timezone.now() - timedelta(days=1),
```

Comment:
Is the `first_seen` here necessary for the test? Just checking to make sure since other than this the tests look identical.